lib/text_diff.ml

```diff
@@ -41,6 +41,23 @@ let md_to_lines name md =
   let module_str = "module " ^ name ^ ": " ^ Buffer.contents buf in
   CCString.lines module_str
 
+let mtd_to_lines name mtd =
+  let buf = Buffer.create 256 in
+  let formatter = Format.formatter_of_buffer buf in
+  match mtd.mtd_type with
+  | Some m ->
+      Printtyp.modtype formatter m;
+      Format.pp_print_flush formatter ();
+      let module_type_str =
+        "module type " ^ name ^ " = " ^ Buffer.contents buf
+      in
+      CCString.lines module_type_str
+  | None ->
+      Printtyp.modtype_declaration (Ident.create_local name) formatter mtd;
+      Format.pp_print_flush formatter ();
```

Comment:
This is unused in this branch, you can remove those lines!

Reply:
You can also move the buffer and formatter bindings to the `| Some m ->` branch since they are not used here either!